packages/contentful/migrations/news/20221028130957-add-fields-to-news-content-type.js

```diff
@@ -0,0 +1,154 @@
+module.exports.description = 'Adds fields in news content type.';
+
+module.exports.up = function (migration) {
+  const news = migration.editContentType('news');
+  news.deleteField('id');
```

Comment:
don't we need an ID?

Reply:
I wasn't sure about this. I thought we could use this id generated by Contentful (in "sys") or we want to copy the id from Squidex when we migrate the data?

![Screen Shot 2022-10-28 at 08 41 38](https://user-images.githubusercontent.com/16595804/199034078-74550de1-ea09-4ca0-9893-e2afc6128896.png)

Reply:
well for News we probably don't need to but I think eventually we are going to have to use a unique ID which is the same as in squidex. Otherwise after the migration all urls will change - question is do we care?